src/layout/Layout.jsx

```diff
@@ -0,0 +1,19 @@
+import { useGetSampleUser } from "../hooks/useGetSampleUser";
+import Footer from "../components/Footer/Footer";
+import NavBar from "../components/NavBar/NavBar";
+
+const Layout = ({ children }) => {
+  const { data } = useGetSampleUser();
+  const { email, profileImageSource } = data || {};
+  const profile = data ? { email, profileImageSource } : null;
+
+  return (
+    <div>
+      <NavBar profile={profile} />
```

Comment:
P2: 
위에서 email, profileImageSource를 구조분해할당했는데 다시 처리할 필요가 있나요?
```suggestion

  return (
    <div>
      <NavBar profile={{ email, profileImageSource }} />
```